src/components/WhatsYellow/index.js

```diff
@@ -48,7 +48,6 @@ export default function WhatsYellow() {
                             );
                         })}
                     </div>
-                    <img className="section__center-img" src={useBaseUrl('/img/evolution.png')} />
```

Comment:
please delete this image for now

Reply:
it's in use on 404 page on Figma. Delete it anyway?

Reply:
Done